### ISSUE TITLE
Bento check fails when no git email configured

### ISSUE THREAD

**Issue by embailie** [returntocorp/bento#107]
I am running version bento/0.1.1b4

**Describe the bug**
I do not have a global email set for my git configuration. 
When I run `bento check` without first running `bento init`, I encounter a failure stacktrace and the CLI fails out. 

**Expected result**
I can execute a bento check. 

**Actual result**
```
bento check
Welcome to Bento! You're about to get a powerful suite of tailored tools.

To get started for the first time, please review our terms of service:

│ Bento does not collect any source code or send any source code outside of your
│ computer. Bento collects usage data to help us understand how to improve the
│ product by aggregating anonymized information about errors, slow operations,
│ feature utilization, and user preferences. We do not and will not sell or share
│ this data to any third party. For more information, see:
│ https://github.com/returntocorp/bento/blob/master/PRIVACY.md

Do you agree to Bento's terms of service and privacy policy? [Y/n]: Y
For the Bento beta, may we contact you infrequently via email to ask for your feedback and let you know about updates? You can unsubscribe at any time. [Y/n]: Y
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/bento/__main__.py", line 14, in main
    cli()
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1134, in invoke
    Command.invoke(self, ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/bento/cli.py", line 443, in cli
    registered = register_user()
  File "/usr/local/lib/python3.7/site-packages/bento/cli.py", line 388, in register_user
    "Email", type=str, default=bento.metrics.__get_git_user_email()
  File "/usr/local/lib/python3.7/site-packages/bento/metrics.py", line 23, in __get_git_user_email
    return r.config_reader().get_value("user", "email")
  File "/usr/local/lib/python3.7/site-packages/git/config.py", line 590, in get_value
    valuestr = self.get(section, option)
  File "/usr/local/lib/python3.7/site-packages/git/config.py", line 79, in assure_data_present
    return func(self, *args, **kwargs)
  File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/configparser.py", line 780, in get
    d = self._unify_values(section, vars)
  File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/configparser.py", line 1146, in _unify_values
    raise NoSectionError(section) from None
configparser.NoSectionError: No section: 'user'
```

**Comment by arashd** [comment]
I'm still seeing this when I unset my global git email address:

```bash
$ git config --global --unset user.email
$ bento init
Welcome to Bento! You're about to get a powerful suite of tailored tools.

To get started for the first time, please review our terms of service:

│ Bento does not collect any source code or send any source code outside of your
│ computer. Bento collects usage data to help us understand how to improve the
│ product by aggregating anonymized information about errors, slow operations,
│ feature utilization, and user preferences. We do not and will not sell or share
│ this data to any third party. For more information, see:
│ https://github.com/returntocorp/bento/blob/master/PRIVACY.md

Do you agree to Bento's terms of service and privacy policy? [Y/n]:
For the Bento beta, may we contact you infrequently via email to ask for your feedback and let you know about updates? You can unsubscribe at any time. [Y/n]:
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/configparser.py", line 788, in get
    value = d[option]
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/collections/__init__.py", line 914, in __getitem__
    return self.__missing__(key)            # support subclasses that define __missing__
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/collections/__init__.py", line 906, in __missing__
    raise KeyError(key)
KeyError: 'email'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/bento/__main__.py", line 14, in main
    cli()
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1134, in invoke
    Command.invoke(self, ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/bento/cli.py", line 443, in cli
    registered = register_user()
  File "/usr/local/lib/python3.7/site-packages/bento/cli.py", line 388, in register_user
    "Email", type=str, default=bento.metrics.__get_git_user_email()
  File "/usr/local/lib/python3.7/site-packages/bento/metrics.py", line 29, in __get_git_user_email
    return r.config_reader().get_value("user", "email")
  File "/usr/local/lib/python3.7/site-packages/git/config.py", line 590, in get_value
    valuestr = self.get(section, option)
  File "/usr/local/lib/python3.7/site-packages/git/config.py", line 79, in assure_data_present
    return func(self, *args, **kwargs)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/configparser.py", line 791, in get
    raise NoOptionError(option, section)
configparser.NoOptionError: No option 'email' in section: 'user'
✘ There was an exception No option 'email' in section: 'user'
$ bento --version
bento/0.1.1b6
```

**Comment by dlukeomalley** [comment]
@arashd Are you able to reproduce this with the 0.2.* releases?

**Comment by arashd** [comment]
I can confirm it's fixed in 0.2.*
